theano/tensor/nnet/tests/test_abstract_conv.py

```diff
@@ -134,9 +134,8 @@ def run_fwd(self, inputs_shape, filters_shape, ref=conv_corr,
         if target_op is not None:
             assert any([isinstance(n.op, target_op) for n
                         in f.maker.fgraph.toposort()])
-            if check_trace:
```

Comment:
why all check_trace are removed? @abergeron did you do that? @slefrancois don't recall doing that. We can't remove all the check_trace check. We need to keep them. So even if it was wanted to remove them, now it must be added back.

This PR also need a rebase.